packages/suite/src/views/guide/FeedbackTypeSelection.tsx

```diff
@@ -147,13 +147,13 @@ const FeedbackTypeSelection = () => {
                     <DetailItem>
                         <Translation id="TR_APP" />
                         :&nbsp;
-                        {appUpToDate ? (
+                        {appUpToDate && isProd() ? (
                             <>
                                 <StyledIcon icon="CHECK" size={10} />
                                 <Translation id="TR_UP_TO_DATE" />
                             </>
                         ) : (
-                            process.env.VERSION
+                            `${process.env.VERSION}-dev`
```

Comment:
this will add `-dev` suffix for every version, that is not up-to-date even if it is production build...

Reply:
fixed